ma_gym/__init__.py

```diff
@@ -40,11 +40,12 @@
         kwargs={'n_agents': 4, 'full_observable': observability, 'step_cost': -0.1}
     )
 
-    register(
-        id='TrafficJunction-v' + str(i),
-        entry_point='ma_gym.envs.traffic_junction:TrafficJunction',
-        kwargs={'full_observable': observability}
-    )
+    for num_max_cars in [4, 10]:
+        register(
+            id='TrafficJunction{}-v'.format(num_max_cars) + str(i),
```

Comment:
Also, the name of env. has changed now, It will be either called TrafficJunction4-v0 or TrafficJunction10-v0 based on the number of maximum cars allowed.